examples/experimental/dagster-airlift/dagster_airlift/core/airflow_defs_data.py

```diff
@@ -38,23 +42,24 @@ def all_dag_ids(self) -> AbstractSet[str]:
     def asset_keys_per_task_handle(self) -> Mapping[TaskHandle, AbstractSet[AssetKey]]:
         asset_keys_per_handle = defaultdict(set)
         for spec in self.resolved_airflow_defs.get_all_asset_specs():
-            if is_mapped_asset_spec(spec):
+            if is_task_mapped_asset_spec(spec):
                 task_handles = task_handles_for_spec(spec)
                 for task_handle in task_handles:
                     asset_keys_per_handle[task_handle].add(spec.key)
         return asset_keys_per_handle
 
     @cached_property
-    def asset_key_per_dag(self) -> Mapping[str, AssetKey]:
-        dag_id_to_asset_key = {}
+    def asset_keys_per_dag(self) -> Mapping[str, AbstractSet[AssetKey]]:
```

Comment:
requires pluralization in AirflowDefinitionsData, since we can now have arbitrary numbers of asset keys per dag

Reply:
And specifically because we have assets that are targeted by multiple dags, correct?

Reply:
nope it's a side effect of providing an arbitrary definitions object. consider the following:
```python
standalone_dag_defs(
    "dag_id",
    Definitions([AssetSpec("1"), AssetSpec("2")])
)
```